app.css

```diff
@@ -36,3 +36,37 @@ body {
 .head-wrapper header ul li:hover {
     border-bottom: 1px solid #4cc5e2;
 }
+
+/*Slick*/
+.single-item {
+  direction: ltr;
+  margin: auto;
+  margin-top:80px;
```

Comment:
I dont understand, how can i do PR for lines 30-40 for example, without including lines 20-30?